### PR TITLE
fix durbinWatsonTest tidier to return a tibble

### PR DESCRIPTION
**Pull request by mkuehn10**
addendum to #266

### CODE REVIEW

File: R/durbinWatsonTest_tidiers.R [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #' Tidies Durbin-Watson Test objects, from the \code{car} package
 #' into a one-row data frame.
 #'
-#' @param x An object of class \code{"durbinWatsonTest"}
+#' @param x An object of class \code{durbinWatsonTest}
 #' @param ... extra arguments (not used)
 #'
```
Comment (alexpghayes) [inline R/durbinWatsonTest_tidiers.R]: Just switched over to documenting with rmarkdown, this seems like a good idea to fix now as well: [`rmarkdown`](https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html).

File: R/durbinWatsonTest_tidiers.R [hunk 2]
```diff
@@ -25,7 +25,8 @@ NULL
 #' @rdname durbinWatsonTest_tidiers
 #' @export
 tidy.durbinWatsonTest <- function(x, ...) {
-  fix_data_frame(
+  df <- 
+    fix_data_frame(
```
Comment (alexpghayes) [inline R/durbinWatsonTest_tidiers.R]: It's only necessary to call `fix_data_frame` on `data.frame` objects. In this case it's better to just directly create a `tibble`.

File: R/durbinWatsonTest_tidiers.R (hunk 1 above)
Comment (mkuehn10) [reply R/durbinWatsonTest_tidiers.R]: I'm not sure what needs to be done for this.

Comment (alexpghayes) [reply R/durbinWatsonTest_tidiers.R]: Instead of using \code{car}, use `car`. It's  a different way to markup the documentation.

**Pull request merged**